nixos/tests/btrbk-section-order.nix

```diff
@@ -0,0 +1,43 @@
+import ./make-test-python.nix ({ lib, pkgs, ... }: {
```

Comment:
a reference to the issue and an explanation of what order is wrong and what order is ok would be nice. Especially as this test is whitespace sensitive, so we will probably have to bless changes in the future and at this point we will need to know what exactly is ok and what is not.